include/deal.II/lac/arpack_solver.h

```diff
@@ -238,6 +246,17 @@ class ArpackSolver : public Subscriptor
   bool initial_vector_provided;
   std::vector<double> resid;
 
+  /**
+   * Real part of the shift
+   */
+  double sigmar;
```

Comment:
I know, I looked it up how `dnaup` takes the argument. But we can still do it differently if we think that would be nicer :-)

I'm not set on it. Do as you think is right.